### ISSUE TITLE
Sidekick WebIde

### ISSUE THREAD

**Issue by georgearmenta** [runsidekick/sidekick#82]
Hello @serkan-ozal @suleymanbarman @odayibasi @rwxdash @boroskoyo 

First of all I am sorry about the discontinue of your product.

Is there a specific reason not to continue your product?

Are you able to open source the web ide?

**Comment by manizm** [comment]
I don't think that is going to happen: https://www.catchpoint.com/press-releases/catchpoint-invests-to-advance-api-cloud-functions-and-microservices-monitoring
